README.md

```diff
@@ -8,14 +8,14 @@ For general contribution process see [CONTRIBUTING.md](./CONTRIBUTING.md)
 
 ## oneAPI DPC++ compiler
 
-[![](https://spec.oneapi.io/oneapi-logo-white-scaled.jpg)](https://www.oneapi.io/)
+[![oneAPI logo](https://spec.oneapi.io/oneapi-logo-white-scaled.jpg)](https://www.oneapi.io/)
 
 [![SYCL Post Commit](https://github.com/intel/llvm/actions/workflows/sycl_post_commit.yml/badge.svg?branch=sycl)](https://github.com/intel/llvm/actions/workflows/sycl_post_commit.yml)
 [![Generate Doxygen documentation](https://github.com/intel/llvm/actions/workflows/gh_pages.yml/badge.svg?branch=sycl)](https://github.com/intel/llvm/actions/workflows/gh_pages.yml)
 
 The DPC++ is a LLVM-based compiler project that implements compiler and runtime
 support for the SYCL\* language. The project is hosted in the
-[sycl](/../../tree/sycl) branch and is synced with the tip of the LLVM upstream
+[sycl](https://github.com/intel/llvm/tree/sycl) branch and is synced with the tip of the LLVM upstream
```

Comment:
Could you try to keep 80-character limit, please?

Reply:
I actually pride myself on that rule... In this MarkDown example, the rule is satisfied when the MarkDown is rendered...

But if we want to apply the rule on raw MarkDown, no problem (change coming up).